pkg/gofr/config/godotenv.go

```diff
@@ -64,6 +66,14 @@ func (e *EnvLoader) read(folder string) {
 	} else {
 		e.logger.Infof("Loaded config from file: %v", overrideFile)
 	}
+
+	// Reload system environment variables to ensure they override any previously loaded values
+	for _, envVar := range os.Environ() {
+		keyValue := strings.SplitN(envVar, "=", splitParts)
+		if len(keyValue) == splitParts {
```

Comment:
I don't think you will get a number bigger than 2 with SplitN 2

Is the idea about testing at least equal was present?

Reply:
Updated this code to this:
```go

	// Reload system environment variables to ensure they override any previously loaded values
	for _, envVar := range os.Environ() {
		key, value, found := strings.Cut(envVar, "=")
		if found {
			os.Setenv(key, value)
		}
	}
```